### DISCUSSION TITLE
disable auto on top when shrinked ( in parameters )

### DISCUSSION THREAD

**Discussion by SanshooSenshi** [WindowTop/WindowTop-App#174] [Ideas]
disable auto on top when shrinked ( in parameters ) 

When I used the shrink resized, if I used it on click ( it goes back to fullscreen in my case ) if I put shrink again it will always set On Top by default... could be usefull in we could unset it in parameters .

**Comment by gileli121** [comment]
@SanshooSenshi 
If the window will not be on top by default when it is in Shrink mode, you will have a little hard time to pick it because it is not visible in the taskbar or alt+tab selection.

If you still want this feature I suggest that when disabling the top mode, it will be visible in the taskbar and alt+tab selection

**Reply by SanshooSenshi** [reply]
If you think it's necessary ... I was thinking It will just put the window on the back of every other windows , so I just have to minimise them to see it back ... but maybe it will be easier to get it right from the taskbar too ...
